css/input.css

```diff
@@ -0,0 +1,124 @@
+
+
+.login_bar {
+	width: 300px;
+	height:45px;
+	background-color:#6B90D9;
+	border-radius: 10px;
+	position: relative;
+    z-index: 1;
+	overflow: hidden;
+	
+	
+}
+
+.login {
+	background-color:#6B90D9;
+	color:#fff;
+	position: absolute;
+	height: 103%;
+	width: 202px;
+	padding-left: 10px;
+	text-align: left;
+	line-height: 50px;
+	vertical-align: middle;
+	-webkit-transition: transform 0.2s ease-in-out 0s;
+	   -moz-transition: transform 0.2s ease-in-out 0s;
+	     -o-transition: transform 0.2s ease-in-out 0s;
+	        transition: transform 0.2s ease-in-out 0s;
+}

```

Comment:
change the color of the placeholder by selecting **::placeholder** the contrast between placeholder and the background is realy small, you can not read it properly

Reply:
done! i changed it to black